src/pymor/reductors/symplectic.py

```diff
@@ -0,0 +1,99 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+from numbers import Number
+
+import numpy as np
+from pymor.algorithms.projection import project
+from pymor.algorithms.simplify import contract, expand
+from pymor.algorithms.symplectic import SymplecticBasis
+from pymor.core.base import BasicObject
+from pymor.models.symplectic import (BaseQuadraticHamiltonianModel,
+                                     QuadraticHamiltonianModel)
+from pymor.operators.constructions import IdentityOperator
+from pymor.operators.numpy import NumpyMatrixOperator
+
+
+class QuadraticHamiltonianRBReductor(BasicObject):
+    """Symplectic Galerkin projection of a |QuadraticHamiltonianModel|.
```

Comment:
That substitution isn't defined in `docs/source/substitutions.py` and makes [docs build fail](https://zivgitlab.uni-muenster.de/pymor/pymor/-/jobs/1017414#L3793). Can you either add the substitution or just use the full reference here?

Reply:
Did that.